theme/icons/heading3.svg

```diff
@@ -0,0 +1 @@
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg"><g fill="#000" fill-rule="evenodd"><path d="M1.074 17c-.294 0-.547-.109-.758-.326A1.066 1.066 0 0 1 0 15.907V4.093c0-.294.105-.55.316-.767C.527 3.109.78 3 1.074 3h.345c.294 0 .55.109.767.326.217.217.326.473.326.767v4.354c0 .102.05.153.153.153H7.63c.102 0 .153-.051.153-.153V4.093c0-.294.109-.55.326-.767A1.05 1.05 0 0 1 8.876 3h.328c.294 0 .546.109.757.326.21.217.316.473.316.767v11.814c0 .294-.105.55-.316.767-.21.217-.463.326-.757.326h-.328a1.05 1.05 0 0 1-.767-.326 1.05 1.05 0 0 1-.326-.767v-5.082c0-.115-.05-.173-.153-.173H2.665c-.102 0-.153.058-.153.173v5.082c0 .294-.109.55-.326.767a1.05 1.05 0 0 1-.767.326h-.345zM13.4 14.856c.162-.251.34-.328.531-.229.06.03.156.087.289.174a3.783 3.783 0 0 0 .99.447 3.73 3.73 0 0 0 1.76.037c.231-.06.44-.153.628-.281.424-.29.636-.697.636-1.22 0-.483-.22-.857-.658-1.123-.414-.247-.954-.37-1.619-.37-.3 0-.52.012-.658.037-.172.03-.286.044-.34.044-.054 0-.108-.034-.163-.103a.762.762 0 0 1-.125-.244 1.562 1.562 0 0 1-.082-.462c0-.136.021-.247.063-.333.042-.086.132-.144.27-.174 1.434-.32 2.339-.79 2.713-1.412a1.36 1.36 0 0 0 .214-.724c0-.251-.043-.455-.129-.61a1.15 1.15 0 0 0-.351-.388c-.296-.212-.658-.318-1.087-.318-.34 0-.626.064-.857.192-.429.237-.68.52-.754.85-.06.232-.211.348-.455.348-.244 0-.432-.021-.565-.063a.87.87 0 0 1-.31-.159c-.12-.108-.178-.277-.178-.506 0-.23.048-.451.144-.666.096-.214.226-.406.392-.576.165-.17.356-.32.573-.447.216-.129.446-.234.687-.318a4.113 4.113 0 0 1 1.4-.259c.457 0 .878.054 1.265.163.387.108.73.273 1.031.495.67.493 1.006 1.175 1.006 2.048 0 .724-.281 1.343-.843 1.855-.39.35-.877.624-1.464.82.784 0 1.43.237 1.937.71.488.459.732.98.732 1.564 0 .584-.105 1.077-.314 1.478a3.24 3.24 0 0 1-.817 1.017 3.682 3.682 0 0 1-2.395.85c-1.36 0-2.408-.303-3.142-.91-.237-.187-.355-.338-.355-.454 0-.116.027-.216.081-.3l.318-.48z"/></g></svg>
```

Comment:
Why is h3 icon that much bigger than H1 and H2? Didn't you miss some compression?

Reply:
I've compressed it via [SVGO](https://jakearchibald.github.io/svgomg/). The reason of bigger size than h1 & h2, because of the complicated path of "3" I guess.